### PR TITLE
Fix the typos in php doc

### PR DESCRIPTION
**Pull request by volha-pivavarchyk**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [yes]
| New feature/enhancement? (use the a.x branch)      | [no]
| Deprecations?                          | [no]
| BC breaks? (use the c.x branch)        | [no]
| Automated tests included?              | [no] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

This PR fixes the typos in php docs. These typos affect phpstan analysis.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. There is no change in functionality, so just make sure the tests work well and code review

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11312"></a>

### CODE REVIEW

**Comment by jos0405** [comment]
Not sure if it's relevant, but email images not loading in grapesjs:
image

**Comment by jos0405** [comment]
The testing steps are really broad - what exactly am I looking for?

**Comment by volha-pivavarchyk** [comment]
@jos0405 Thanks! You are right, it is really broad. I changed the steps to test

File: app/bundles/CoreBundle/Event/CustomTemplateEvent.php [hunk 1]
```diff
@@ -63,11 +63,9 @@ public function getVars()
     }
 
     /**
-     * @param arrat $vars
-     *
-     * @return CustomTemplateEvent
+     * @param array<string, mixed> $vars
      */
-    public function setVars($vars)
+    public function setVars(array $vars): CustomTemplateEvent
```
Comment (escopecz) [inline app/bundles/CoreBundle/Event/CustomTemplateEvent.php]: Adding types to existing properties or return types in public or protected methods is considered a BC break.  Please improve the types in the method comments only.

Comment (volha-pivavarchyk) [reply app/bundles/CoreBundle/Event/CustomTemplateEvent.php]: Thanks! I've fixed it

**Review by escopecz — APPROVED** [review]
Thanks! I'll merge this without additional review because it does not touch any code. Just comments.

**Pull request merged**